src/library/parsercsv.h

```diff
@@ -20,21 +7,22 @@
 #include "library/parser.h"
 #include "library/basesqltablemodel.h"
 
-class ParserCsv : public Parser
-{
+class ParserCsv : public Parser {
     Q_OBJECT
-public:
+  public:
     ParserCsv();
```

Comment:
The default constructor declaration is probably redundant, see my comment. `~ParserCsv() override = default;` would be more appropriate here.